airbyte-ci/connectors/pipelines/README.md

```diff
@@ -555,6 +557,30 @@ Migrate connectors the poetry package manager.
 Migrate source-openweather to use the base image:
 `airbyte-ci connectors --name=source-openweather migrate-to-poetry`
 
+### <a id="connectors-migrate_to_inline_schemas"></a>`connectors migrate_to_inline_schemas` command
+
+Migrate .json schemas into manifest.yaml files, when present.
+
+#### Examples
+
+Migrate source-quickbooks to use inline schemas:
+`airbyte-ci connectors --name=source-quickbooks migrate_to_inline_schemas`
+
+### <a id="connectors-pull_request"></a>`connectors pull_request` command
+
+Makes a pull request for all changed connectors. If the branch already exists, it will update the existing one.
+
+#### Examples
+
+Make a PR for all changes, bump the version and make a changelog in those PRs. They will be on the branch ci_update/round2/<connector-name>:
```

Comment:
```suggestion
Make a PR for all changes, bump the version and make a changelog in those PRs. They will be on the branch `ci_update/round2/<connector-name>`:
```

Reply:
Very curious why `round2` here. Intentional? (haven't read the rest of the PR yet)